appcues/src/debug/java/com/appcues/ExperienceTestHelper.kt

```diff
@@ -15,11 +17,13 @@ public fun Appcues(
     accountId: String,
     applicationId: String,
     imageLoader: ImageLoader?,
+    coroutineScope: CoroutineScope?,
     config: (AppcuesConfig.() -> Unit)? = null,
 ): Appcues = Bootstrap.createScope(
     context = context,
-    config = AppcuesConfig(accountId, applicationId).apply {
-        config?.invoke(this)
-        this.imageLoader = imageLoader
-    }
-).get()
+    config = AppcuesConfig(accountId, applicationId).apply { config?.invoke(this) }
+).also { scope ->
+    // optional test overrides
+    imageLoader?.let { scope.define(ImageLoader::class, ScopedDefinition { imageLoader }, true) }
+    coroutineScope?.let { scope.define(CoroutineScope::class, ScopedDefinition { coroutineScope}, true) }
```

Comment:
this is test only code (in `src/debug` folder) -- allows the tests to pass in a CoroutineScope, in addition to the ImageLoader option we had before. Now, it uses a new `override: true` option on the `scope.define` call that I added. This allows our normal code to be totally unaware of these test specifics, and just do the override right here.